src/gateway/get-started/proxy-caching.md

```diff
@@ -1,73 +1,190 @@
+book: get-started
+chapter: 4
 ---
 
+One of the ways Kong delivers performance is through caching. 
+The [Proxy Cache plugin](/hub/kong-inc/proxy-cache/) accelerates performance by caching
+responses based on configurable response codes, content types, and request methods.
+When caching is enabled, upstream services are not bogged down with repetitive requests, 
+because {{site.base_gateway}} responds on their behalf with cached results. Caching can be 
+enabled on specific {{site.base_gateway}} objects or for all requests globally.
+
+### Cache Time To Live (TTL)
+
+TTL governs the refresh rate of cached content, which is critical to understand to ensure 
+that clients aren't served outdated content. A TTL of 30 seconds means content older than 
+30 seconds is deemed expired and will be refreshed on subsequent requests. 
+TTL configurations should be set differently based on the type of the content the upstream 
+service is serving. 
 
-One of the ways Kong delivers performance is through caching. The proxy caching  plug-in delivers fast performance by providing the ability to cache responses based on requests, response codes, and content type. This means the upstream services are not bogged down with repeated requests, because {{site.base_gateway}} can respond with cached results. {{site.base_gateway}} offers the [Proxy Caching plugin](/hub/kong-inc/proxy-cache/). The Proxy Caching plugin provides this fast performance using a reverse proxy cache implementation. It caches response entities based on the request method, configurable response code, content type, and can cache per consumer or per API. With that level of granularity you can create caching rules that match your specific use case. 
+* Static data that is rarely updated can have longer TTL
 
+* Dynamic data should use shorter TTL to avoid serving outdated values 
 
+{{site.base_gateway}} follows [RFC-7234 section 5.2](https://tools.ietf.org/html/rfc7234)
+for cached controlled operations. See the specification and the Proxy Cache 
+plugin [parameter reference](/hub/kong-inc/proxy-cache/#parameters) for more details on TTL configurations.
 
-## Configure Proxy Cache plugin
+## Enable caching
 
-Configuring the Proxy Caching plugin is done by sending a `POST` request to the admin API and describing the caching rules:
+The following tutorial walks through managing proxy caching across various aspects in {{site.base_gateway}}.
 
+### Prerequisites
 
-```sh
+This chapter is part of the *Get Started with Kong* series. For the best experience, it is recommended that you follow the
+series from the beginning. 
 
-curl -i -X POST http://localhost:8001/plugins \
-  --data name=proxy-cache \
-  --data config.content_type="application/json" \
-  --data config.cache_ttl=30 \
-  --data config.strategy=memory
+Start with the introduction [Get Kong](/gateway/latest/get-started/get-kong), which includes
+tool prerequisites and instructions for running a local {{site.base_gateway}}.
```

Comment:
```suggestion
a list of prerequisites and instructions for running a local {{site.base_gateway}}.
```

Reply:
this will require updating other pages to match.  I think we should use an include here, i just haven't had the time to figure it out

Reply:
Sounds good.